composer.json

```diff
@@ -141,7 +141,8 @@
   },
   "conflict": {
     "codeception/stub": ">=2.0.2",
-    "symfony/dependency-injection": ">=3.4.15"
+    "symfony/dependency-injection": "3.4.15",
+    "symfony/symfony": "3.4.16"
```

Comment:
are these bugs in all symfony of just like in the example above for dependency-injection ?

Reply:
in the end I marked as conflicting 3.4.15 and 3.4.16